cap/modules/repos/views.py

```diff
@@ -45,7 +45,12 @@ def get_webhook_event():
     payload = request.get_json()
     payload.update(request.headers)  # info about type of event inside headers
 
-    serializer = payload_serializer_factory()
+    # only github sends ping event
+    if 'X-Github-Event' in payload.keys() \
+            and payload['X-Github-Event'] == 'ping':
+        return jsonify({'message': 'Got it.'})
```

Comment:
The reason why we use a payload_serializer_factory or git api factory is that we dont want to hardcode ifs and elses in our methods or endpoints. Try this:
```
github_event_to_serializer = {
    'push': GitHubPushPayloadSchema(),
    'release': GitHubReleasePayloadSchema(),
    'ping': GitHubPingPayloadSchema()
}
```
and then our ping event class:
```
class GitHubPingPayloadSchema(GitHubPayloadSchema):
    """Serializer for a GitLab webhook payload, from a `push` event."""
    event_type = fields.Str(default='ping', dump_only=True)
```
and here:
```
if data["event_type"] == 'ping':
        return jsonify({'message': 'Got it.'})
```